CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.11.0
+
+* Added an optional `postage` argument to `post_precompiled_letter` method and `send_precompiled_letter` that calls it.
```

Comment:
People using the gem probably won't know anything about `post_precompiled_letter` (unless they've been reading the source code), so maybe we should not mention that method in case it's confusing?

Reply:
good call 👍